pl_engine/pl_server/src/main/java/com/cubrid/plcsql/compiler/CoercionScheme.java

```diff
@@ -360,6 +360,9 @@ public abstract List<TypeSpec> getCoercions(
     static {
         compOpCommonType[TypeSpecSimple.IDX_NULL][TypeSpecSimple.IDX_NULL] = TypeSpecSimple.NULL;
 
+        compOpCommonType[TypeSpecSimple.IDX_OBJECT][TypeSpecSimple.IDX_OBJECT] =
+                TypeSpecSimple.OBJECT;
+
```

Comment:
What is the below (line number 368) same code?
```
compOpCommonType[TypeSpecSimple.IDX_OBJECT][TypeSpecSimple.IDX_OBJECT] =
                TypeSpecSimple.OBJECT;
```

Reply:
It's a duplicate by mistake. Let me erase the line. 
Thank you.